src/components/Carousel.vue

```diff
@@ -11,106 +11,105 @@
 </template>
 
 <script>
+import {ref,onMounted} from '@vue/runtime-core'
```

Comment:
Out of curiosity, why do you import from `@vue/runtime-core` instead of `vue`? like this
```js
import { ref } from 'vue'
```

Reply:
It was just to do with how my project was setup, I've updated it now.